### PR TITLE
Persistence

### PR DESCRIPTION
**Pull request by elimisteve**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,14 +61,23 @@ To install and build static assets:
 
 ``` $ npm run dev  # watch asset files and recompile when changed ```
 
-Then, to build and run the go binary
+Then, to set up the database and run PostgREST, which our Go code uses for persistence:
+
+``` $ cd db/ ```
+
+``` $ sudo su postgres -c 'bash init_sql.sh' ```
```
Comment (jimmcgaw) [inline README.md]: FWIW, this didn't work on OS X. (Not saying this shouldn't be here, but it blocked me on getting started doing this.)

Comment (elimisteve) [inline README.md]: Hmm okay, good to know. The key is becoming the `postgres` user then running `bash init_sql.sh`.

File: pg_types.go [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+		return fmt.Errorf(
+			"Got HTTP %v from PostgREST, wanted %v. Resp: %#v (err unmarshal: %v)",
+			resp.StatusCode, statusWanted, respjson, err)
+	}
+
+	return nil
+}
+
+func (cl *PGClient) Get(urlSuffix string) (*http.Response, error) {
+	log.Debugf("GET'ing from %s", urlSuffix)
```
Comment (jimmcgaw) [inline pg_types.go]: Debug? Okay to leave this in?

**Comment by jimmcgaw** [comment]
Looks good! I don't know go well enough to do a proper review, but I didn't see any massive red flags.

File: pg_types.go (hunk 2 above)
Comment (elimisteve) [inline pg_types.go]: A few weeks ago, @ajvb set up `logrus`, so we now have log levels. That statement will only print to stdout if we're not in production, aka when we're dev'ing (yaaay)

**Pull request merged**